src/lib.rs

```diff
@@ -50,6 +50,9 @@ mod verifier;
 ///   - Unknown eBPF helper index.
 pub type Verifier = fn(prog: &[u8]) -> Result<(), Error>;
 
+/// eBPF Jit-compiled program.
+pub type JitCompiled = unsafe fn(*mut u8, usize, *mut u8, usize, usize, usize) -> u64;
```

Comment:
Not necessarily. But it's true that would be more consistent. Hmm... If you haven't done the change already, maybe use simply `JitProgram` instead of `JitCompiled`? Sounds more like a noun and less like an attribute. And then keep `Verifier` untouched? What do you think?